src/server/profiles/allergyintolerance/allergyintolerance.config.js

```diff
@@ -14,6 +14,17 @@ const scopes = [
 	'allergyintolerance/*.read'
 ];
 
+const write_scopes = [
```

Comment:
guess theres no really pretty way to separate this out...

Reply:
yea not really, we can pull them out into their own separate module but then have to pass strings in for each module, so not sure it is much better, may save a couple of lines though.

Reply:
How would you feel about something like this:

```javascript
// In a common.scopes.js file
module.exports.read_scopes = profile => {
	let profile_lower = profile.toLowerCase();

	return [
		'user/*.*',
		`user/${profile_lower}.*`,
		`user/${profile_lower}.read`,
		'user/*.read',
		`${profile_lower}/*.*`,
		`${profile_lower}/${profile}.*`,
		`${profile_lower}/${profile}.read`,
		`${profile_lower}/*.read`
	];
};

// In allergyintolerance.config.js
const read_scopes = common_scopes.read_scopes('AllergyIntolerance');
```